pkg/virt-controller/watch/vm.go

```diff
@@ -1019,3 +1070,15 @@ func (c *VMController) resolveControllerRef(namespace string, controllerRef *v1.
 	}
 	return vm.(*virtv1.VirtualMachine)
 }
+
+func getVirtualMachineRunStrategy(vm *virtv1.VirtualMachine) virtv1.VirtualMachineRunStrategy {
```

Comment:
this function is configure under the virt-api and  also here under the virt-controller can you maybe move it under the api and just reuse it?